source/ranges.tex

```diff
@@ -6270,7 +6270,7 @@
 \pnum
 \begin{example}
 \begin{codeblock}
-constexpr auto source = "  \t   \t   \t   hello there";
+constexpr decltype(auto) source = "  \t   \t   \t   hello there";
```

Comment:
The best fix is to make it a `string_view`:
```suggestion
constexpr auto source = "  \t   \t   \t   hello there"sv;
```
String literals should never be used as ranges because nobody has ever wanted the null terminator in their range.